todo/src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./App.scss";
+import TodoList from "./components/TodoList";
+import TodoForm from "./components/TodoForm";
+
+function App() {
+  return (
+    <div className="App">
+      <header>
+        <h1>Todo List</h1>
+      </header>
+
+      <TodoList />
+      <TodoForm />
```

Comment:
One thing you could do here is abstract your `TodoList` component, `TodoForm` component and your `Todo List` title into another component named `TodoContainer` to keep everything a bit cleaner.